Editors/TiledEditor/source/TiledLevel.hx

```diff
@@ -74,10 +77,32 @@ class TiledLevel extends TiledMap
 			var imagePath 		= new Path(tileSet.imageSource);
 			var processedPath 	= c_PATH_LEVEL_TILESHEETS + imagePath.file + "." + imagePath.ext;
 			
-			var tilemap:FlxTilemap = new FlxTilemap();
+			// could be a regular FlxTilemap if there are no animated tiles
+			var tilemap = new FlxTilemapExt();
 			tilemap.loadMapFromArray(tileLayer.tileArray, width, height, processedPath,
 				tileSet.tileWidth, tileSet.tileHeight, OFF, tileSet.firstGID, 1, 1);
 			
+			if (tileLayer.properties.contains("animated"))
+			{
+				var tileset = tilesets["level"];
+				var specialTiles:Map<Int, TiledTilePropertySet> = new Map();
+				for (tileProp in tileset.tileProps)
+				{
+					if (tileProp != null && tileProp.animationFrames.length > 0)
+					{
+						specialTiles[tileProp.tileID + tileset.firstGID] = tileProp;
+					}
+				}
+				var tileLayer:TiledTileLayer = cast layer;
```

Comment:
I changed this to an unsafe cast, because previously it was a safe cast, which has some unnecessary runtime overhead -see https://haxe.org/manual/expression-cast.html.

Basically, there's no reason to use safe casts unless you intend to try-catch them / if you know the cast is going to succeed anyway.